modules/server/src/network/types/types.ts

```diff
@@ -33,3 +34,5 @@ export type NetworkAggregation = {
 	bucket_count: number;
 	buckets: Bucket[];
 };
+
+export type NodeConfig = NetworkConfig & { aggregations: { name: string; type: string }[] };
```

Comment:
`NetworkConfig` is input to the setup, `NodeConfig` is the output when relevant data is retrieved.
Distinct types because added `aggregations` has an optional type doesn't make sense - we want to make sure it's present